.travis.yml

```diff
@@ -19,13 +19,16 @@ jobs:
       script: xvfb-run npm run test:e2e:ci
     - stage: test
       script: xvfb-run npm run test:e2e:storybook:ci
+    - stage: deploy
+      if: branch = latest AND type = push
+      script: npm run deploy-storybook -- --ci
```

Comment:
Don't you need to run the `npm run build-storybook` step first, before deploying? To ensure you're uploading the latest version of storybook to GH pages.